exasol/toolbox/tools/test_issue.py

```diff
@@ -0,0 +1,71 @@
+import typer
```

Comment:
Like it looks very good, one small thing though I recommend you are using `keyword arguments` when calling the functionality. This makes it easier for the reader and avoids errors due to changes in parameter order etc.
See also https://exasol.github.io/python-styleguide/guides/idioms/general.html